Assets/Mirror/Runtime/Messages.cs

```diff
@@ -3,110 +3,62 @@
 
 namespace Mirror
 {
-    public interface IMessageBase
-    {
-        void Deserialize(NetworkReader reader);
+    [Obsolete("Implement INetworkMessage instead", true)]
+    public interface IMessageBase { }
 
-        void Serialize(NetworkWriter writer);
-    }
+    [Obsolete("Implement INetworkMessage instead", true)]
```

Comment:
```suggestion
    [Obsolete("Implement INetworkMessage instead. Use extension methods instead of Serialize/Deserialize, see https://github.com/vis2k/Mirror/pull/2318", true)]
```